### PR TITLE
Creating proxy server

### PR DESCRIPTION
**Pull request by DmitriyKrivovyaz**

### CODE REVIEW

File: server/src/http/weatherApi.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+const axios = require('axios');
+require('dotenv').config()
+
+const user = process.env.WEATHER_API_USER;
+const password = process.env.WEATHER_API_PASSWORD;
+const WEATHER_API_URL = process.env.WEATHER_API_URL;
+
+class WeatherApi {
+    async getAccessToken() {
+        const {data} = await axios.post(WEATHER_API_URL + '/authorize/token', {user, password});
```
Comment (AV29) [inline server/src/http/weatherApi.js:11]: try/catch?

File: server/src/routes/index.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+const Router = require('express');
+const weatherApiRouter = require('./weatherApiRouter')
+const router = new Router();
+
+
```
Comment (AV29) [inline server/src/routes/index.js:5]: redundant space?

File: server/src/http/weatherApi.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+        return data;
+    }
+}
+
+
```
Comment (AV29) [inline server/src/http/weatherApi.js:16]: redundant space?

File: server/src/http/weatherApi.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
+// const $host = require('./index')
+const axios = require('axios');
+require('dotenv').config()
```
Comment (AV29) [inline server/src/http/weatherApi.js:3]: Chekc if you really need to require 'dontenv' here, as you already do it on a top level

**Review by anton-vlasik — APPROVED** [review]

**Pull request merged**